app/Providers/RepositoryServiceProvider.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Hedonist\Providers;
+
+use Illuminate\Support\ServiceProvider;
+
+class RepositoryServiceProvider extends ServiceProvider
+{
+    /**
+     * Bootstrap services.
+     *
+     * @return void
+     */
+    public function boot()
+    {
+        //
+    }
+
+    /**
+     * Register services.
+     *
+     * @return void
+     */
+    public function register()
+    {
+        $this->app->bind(\App\Repositories\PlaceCategoryRepositoryInterface::class, \App\Repositories\PlaceCategoryRepository::class);
```

Comment:
@ivanshamis Namespace for both classes is wrong. You have `app/Repositories/Place/PlaceCategoryRepository.php`. `use` statement necessary.